src/types.rs

```diff
@@ -379,144 +468,144 @@ mod type_definition_tests {
             { name: my_int, type: my_int }
          */
         IslType::named("my_int", [IslConstraint::type_constraint(IslTypeRef::named("my_int"))]),
-        TypeDefinition::named("my_int", [Constraint::type_constraint(34)])
+        TypeDefinition::named("my_int", [Constraint::type_constraint(35)])
```

Comment:
We need to figure out a better solution for these tests than using the typeId numbers. It's not maintainable.

I don't expect it to be fixed in this PR, but it needs to be done soon. The more difficult it is to write/maintain tests, the less motivated someone will be to write/maintain tests.

Reply:
I will prioritize this issue.